config/workflows/templates/9/job.xml

```diff
@@ -1,59 +1,58 @@
-# Prepare container working directory 
-containerWorkingDirectory = '-w '+workspaceContainer+' ' 
-# Save pre execution command into magic variable 'preJavaHomeCmd', which is picked up by the node 
-preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName 
+              <![CDATA[
+# Prepare Docker parameters 
+containerName = 'java' 
+dockerRunCommand =  'docker run ' 
+dockerParameters = '--rm ' 
+# Prepare ProActive home volume 
+paHomeHost = variables.get("PA_SCHEDULER_HOME") 
+paHomeContainer = variables.get("PA_SCHEDULER_HOME") 
+proActiveHomeVolume = '-v '+paHomeHost +':'+paHomeContainer+' ' 
+# Prepare working directory (For Dataspaces and serialized task file) 
+workspaceHost = localspace 
+workspaceContainer = localspace 
+workspaceVolume = '-v '+localspace +':'+localspace+' ' 
+# Prepare container working directory 
+containerWorkingDirectory = '-w '+workspaceContainer+' ' 
+# Save pre execution command into magic variable 'preJavaHomeCmd', which is picked up by the node 
+preJavaHomeCmd = dockerRunCommand + dockerParameters + proActiveHomeVolume + workspaceVolume + containerWorkingDirectory + containerName
 ]]>
             </code>
           </script>
         </envScript>
       </forkEnvironment>
       <scriptExecutable>
-        <script>
+        <script
>
           <code language="bash">
-            <![CDATA[
-echo '---------------------- Execute a task inside a Docker Container Example -----------------------------'
-echo '---------------------- Start Task No. 1 ---------------------------------------------------------'
-echo 'This task runs inside a Docker container. It can install new software or change software versions without impacting other software or tasks.'
-echo "Now, lets install git in this container by executing 'apt-get install git -y'."
-echo "We could clone a git repo."
-echo 'Lets pretend we created some statistics about this git repo, we save it in a file (stats.log).'
-echo "In the 'Data Management' tab, the stats.log file is setup to be transferred between tasks."
-echo "This is a statistic, 1,2,3,4,5,6,7,8. Saved in stats.log." > stats.log
+            <![CDATA[
+echo '---------------------- Execute a task inside a Docker Container Example -----------------------------'
+echo '---------------------- Start Task No. 1 ---------------------------------------------------------'
+echo 'This task runs inside a Docker container. It can install new software or change software versions without impacting other software or tasks.'
+echo "Now, lets install git in this container by executing 'apt-get install git -y'."
+echo "We could clone a git repo."
+echo 'Lets pretend we created some statistics about this git repo, we save it in a file (stats.log).'
+echo "In the 'Data Management' tab, the stats.log file is setup to be transferred between tasks."
+echo "This is a statistic, 1,2,3,4,5,6,7,8. Saved in stats.log." > stats.log

```

Comment:
I don't understand this message, as I don't see this command being executed in the script